cypress.config.ts

```diff
@@ -1,9 +1,43 @@
 import { defineConfig } from "cypress";
+import LoginPage = require("./cypress/pages/LoginPage");
+
+let employeeId: string;
+type employee = {
+  id: string,
+  username: string,
+  password: string
+};
+let employeeInfo: employee = {
+  id: "",
+  username: "",
+  password: "",
+}
 
 export default defineConfig({
   e2e: {
     setupNodeEvents(on, config) {
       // implement node event listeners here
+      on("task", {
+        setEmployeeId(id: string): null {
+           employeeInfo.id = id;
+           return null;
```

Comment:
use void instead of null.

Reply:
if I use void , this error will be shown:
The task 'setEmployeeId' returned undefined. You must return a value, null, or a promise that resolves to a value or null to indicate that the task was handled